packages/esm-forms-workspace-app/src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const customFormRepresentation =
+  '(uuid,name,display,encounterType:(uuid,name,viewPrivilege,editPrivilege),version,published,retired,resources:(uuid,name,dataType,valueReference))';
+export const customEncounterRepresentation = `custom:(uuid,encounterDatetime,encounterType:(uuid,name,viewPrivilege,editPrivilege),form:${customFormRepresentation}`;
+
+export const formEncounterUrl = `/ws/rest/v1/form?v=custom:${customFormRepresentation}`;
+export const formEncounterUrlPoc = `/ws/rest/v1/form?v=custom:${customFormRepresentation}&q=poc`;
+
+export const defaultPaginationSize = 10000;
```

Comment:
Shouldn't these just go in the place that we're making the API calls? I don't really understand why we need them here...